src/popup/App.vue

```diff
@@ -117,6 +117,10 @@ body {
   background-color: $color-black;
 }
 
+body.bg {
```

Comment:
the class name probably is too unspecific

Reply:
Agreed. Suggestions?

Reply:
how about `body.header-bg`?

Reply:
or `body.main-bg`?

Reply:
or `body.color-bg-3`?